src/components/ArticlesList.jsx

```diff
@@ -0,0 +1,42 @@
+import { useState, useEffect } from 'react';
+import ArticleCard from './ArticleCard';
+import { getArticles, getUsers } from '../api';
+
+const ArticlesList = () => {
+    const [articles, setArticles] = useState([]);
+//    const [authors, setAuthors] = useState([]);
+    const [isLoading, setIsLoading] = useState(true);
+    const [isError, setIsError] = useState(false);
+
+    useEffect(() => {
+        setIsLoading(true)
+        getArticles()
+        .then((articles) => {
```

Comment:
Try console.loggging to take a look at what's in these articles. I don't think, for the article cards that make up this list, any more information is needed than what you're getting back from /api/articles. It is not necessary to make a request for /api/articles/:article_id down in your comment card, because we already have the data we need. Once we get to a point where we want to look at a single article on it's own with more detail, _then_ you might look to make a call to /api/articles/:article_id